### ISSUE TITLE
fsortperm() throws uint_mapping error under Julia-1.9.0-beta2

### ISSUE THREAD

**Issue by AdamWysokinski** [xiaodaigh/SortingLab.jl#19]
Hi,
With the latest beta of Julia (1.9.0-beta2), fsortperm() does not work properly with vectors of floats:
```
julia> fsortperm(rand(4))
ERROR: MethodError: no method matching uint_mapping(::Base.Order.ForwardOrdering, ::Float64)

julia> fsortperm([1, 3, 2, 4])
4-element Vector{Int64}:
 1
 3
 2
 4
```

**Comment by xiaodaigh** [comment]
Couldn't get vscode working on 1.9 I will get to this when 1.9 works with vscode

**Comment by AdamWysokinski** [comment]
Ok, thanks.

**Comment by xiaodaigh** [comment]
I will  there's no issue since base 1.9 introduced radix sort to base. I will see what I can do here.

**Comment by xiaodaigh** [comment]
fixed. but you don't need this package for sorting floats. only strings would be good. `fsortrperm` might be good for all things though

**Comment by xiaodaigh** [comment]
fixed by #20
